docs/v1/p4runtime-id-notes.md

```diff
@@ -9,19 +9,9 @@ in one place notes on those identifiers, including:
 + When the values are chosen
 + What is their scope in "space" and time in which they must be unique
 
-This document was written while referring to the P4Runtime API
-specification as of this commit of the Github repository
-https://github.com/p4lang/p4runtime
-
-```
-commit c9cd4af17c2d562d6e4ec760237f16fea9558fcb
```

Comment:
How did that get in there?

Reply:
I think at the time @jafingerhut wrote the guide, this was the latest commit on the master branch, and he wanted to document the exact version of the spec covered by the guide. I am removing this now because we can actually use the version tag, and nothing has changed AFAIK since Andy wrote this doc.